metrics/api/src/main/java/io/helidon/metrics/api/Util.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.api;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+class Util {
+
+    private Util() {
+    }
+
+    static <T> Iterable<T> iterable(T[] items) {
+        if (items == null || items.length == 0) {
```

Comment:
This can just be replaced with `List.of(items)`, this method is not needed.

Reply:
At the cost of creating a `List` each time.